test.php

```diff
@@ -13,7 +13,9 @@
 $WPT_TEST_DIR = getenv( 'WPT_TEST_DIR' );
 $WPT_SSH_OPTIONS = getenv( 'WPT_SSH_OPTIONS' ) ? : '-o StrictHostKeyChecking=no';
 $WPT_PHP_EXECUTABLE = getenv( 'WPT_PHP_EXECUTABLE' ) ? : 'php';
-$WPT_PHPUNIT_CMD = getenv( 'WPT_PHPUNIT_CMD' ) ? : 'cd ' . escapeshellarg( $WPT_TEST_DIR ) . '; ' . $WPT_PHP_EXECUTABLE . ' phpunit.phar';
+
+// This uses `||` to run PHPUnit when it is downloaded manually (like for PHP 5.6-7.0) rather than through Composer.
+$WPT_PHPUNIT_CMD = getenv( 'WPT_PHPUNIT_CMD' ) ? : 'cd ' . escapeshellarg( $WPT_TEST_DIR ) . ' && ./vendor/phpunit/phpunit/phpunit || php phpunit.phar';
```

Comment:
I was testing and noticed I missed something here. This needs to use `$WPT_PHP_EXECUTABLE` to call the phar.

Reply:
This looks to be a problem with both methods of running. The test runner will report the version of PHP used to *install* WordPress during the tests (which is accurate with the current PR), and this doesn't necessarily match with the version of PHP that is being used for the tests.

Putting together a patch.

Reply:
I'm back to getting OOM killed in the VM I'm testing with versions other than PHP 7.2.

I'm wondering if this might be specific to my VM, though, rather than an issue with code. Going to go ahead and update the PR, and would appreciate any testing after that.

Reply:
Updated. Would appreciate testing.

Also noticing that this might result in an unexpected event if composer run PHPUnit returns an error, since it might try to run PHPUnit via PHAR after.

That shouldn't cause immediate problems I can think of, because that PHAR shouldn't exist in that case, but worth pointing out in case there's a better approach.